mysql scripts/table_creation/FullDB.sql

```diff
@@ -0,0 +1,172 @@
+    scheduledTime DATETIME NOT NULL,
+	createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    archived BOOLEAN NOT NULL DEFAULT FALSE,
+	clientId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY (clientId) REFERENCES Client(clientId),
+    siteId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY (siteId) REFERENCES Site(siteId)
+);
+
+CREATE TABLE Answer (
+	answerId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+	possibleAnswerId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY (possibleAnswerId) REFERENCES PossibleAnswer(possibleAnswerId),
+    appointmentId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY (appointmentId) REFERENCES Appointment(appointmentId),
+    litmusQuestionId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY (litmusQuestionId) REFERENCES LitmusQuestion(litmusQuestionId)
+);
+
+CREATE TABLE User (
+	userId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+	firstName VARCHAR(65) NULL,
+	lastName VARCHAR(65) NULL,
+	email VARCHAR(355) NOT NULL,
+	phoneNumber VARCHAR(20) NULL,
+	preparesTaxes BOOLEAN NOT NULL DEFAULT FALSE, 
+	archived BOOLEAN NOT NULL DEFAULT FALSE
+);
+
+
+CREATE TABLE ServicedAppointment (
+	servicedAppointmentId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    startTime DATETIME,
+    endTime DATETIME,
+    createdAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    userId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY(userId) REFERENCES User(userId),
+    appointmentId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY(appointmentId) REFERENCES Appointment(appointmentId)
+);
+
+CREATE TABLE Login (
+	failedLoginCount INT UNSIGNED ZEROFILL NOT NULL DEFAULT 0,
+	password VARCHAR(255) NOT NULL,
+	lockoutTime TIMESTAMP NOT NULL DEFAULT 0,
+	userId INTEGER UNSIGNED NOT NULL,
+	FOREIGN KEY(userId) REFERENCES User(userId)
+);
+
+CREATE TABLE LoginHistory (
+	loginHistoryId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+	timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	ipAddress VARCHAR(95) NOT NULL,
+	userId INTEGER UNSIGNED NOT NULL,
+	FOREIGN KEY(userId) REFERENCES User(userId)
+);
+	
+CREATE TABLE PasswordReset (
+	token VARCHAR(255) NOT NULL,
+	ipAddress VARCHAR(95) NOT NULL,
+	timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	archived BOOLEAN NOT NULL DEFAULT FALSE,
+	userId INTEGER UNSIGNED NOT NULL,
+	FOREIGN KEY (userId) REFERENCES User(userId)
+);
+
+CREATE TABLE Privilege (
+	privilegeId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    description VARCHAR(500) NOT NULL,
+    tag VARCHAR(255) NOT NULL,
+	CONSTRAINT uniqueTag UNIQUE INDEX(tag)
+);
+
+CREATE TABLE UserPrivilege (
+	userPrivilegeId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    userId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY(userId) REFERENCES User(userId),
+    privilegeId INTEGER UNSIGNED NOT NULL,
+    FOREIGN KEY(privilegeId) REFERENCES Privilege(privilegeId)
+);
+
+CREATE TABLE Ability (
```

Comment:
I figured name would normally suffice, since it'll always be something like "Can Speak Spanish" or "Basic Certified". But we can totally add a description, I see no harm in doing so.